ESLint/example.eslintrc

```diff
@@ -1,10 +1,10 @@
 // Copy this file to the root directory and modify <path-to>
 
 {
-    "extends": "<path-to>/lombiq-base.js",
+  "extends": "<path-to>/lombiq-base.js",
 
-    "rules": {
-        // Add overriding rules here like this:
-        // "example-rule" : "error"
-    }
+  "rules": {
+    // Add overriding rules here like this:
+    // "example-rule" : "error"
+  }
```

Comment:
Why did the indentation change here and in the lines above? Was that intentional?

Reply:
Yes, it wasn't formatted how JSON files should be, with two spaces (this is the VS default and what I found we actually do use in other JSON files in the Finitive project).